### PR TITLE
Add resources to the init container

### PR DESCRIPTION
**Pull request by AjayTripathy**

### CODE REVIEW

**Comment by dwbrown2** [comment]
Can you share how you tested?

Also, I personally do not feel that the TODO is necessary to include.

**Comment by AjayTripathy** [comment]
Tested by helm upgrading these values with/without a resources: request block. I can remove the TODO

File: cost-analyzer/values.yaml [hunk 1]
```diff
@@ -103,10 +103,6 @@ kubecostModel:
     #  cpu: "800m"
     #  memory: "256Mi"
 
-kubecostInit:
```
Comment (dwbrown2) [inline cost-analyzer/values.yaml:106]: is there any way this can cause errors, i.e. someone still passing this value?

if not then, LGTM

**Review by dwbrown2 — APPROVED** [review]
One question below

File: cost-analyzer/values.yaml (hunk 1 above)
Comment (AjayTripathy) [reply cost-analyzer/values.yaml:106]: Passing extraneous values is fine.

**Pull request merged**